source/language.c

```diff
@@ -12,6 +12,7 @@
 #define STR_PT(_str) [CFG_LANGUAGE_PT] = _str
 #define STR_RU(_str) [CFG_LANGUAGE_RU] = _str
 #define STR_TW(_str) [CFG_LANGUAGE_TW] = _str
+#define STR_NO(_str) [CFG_LANGUAGE_NO] = _str
```

Comment:
I'm assuming CFG_LANGUAGE_NO doesn't exist.

Reply:
That is correct. I wasn't sure what else I could've put in there instead. 😅

Reply:
It doesn't, and therefore the code doesn't compile. Unfortunately we cannot accept languages which don't already exist in Nintendo's language list.